### ISSUE TITLE
[QUESTION] TopK router （k>1）

### ISSUE THREAD

**Issue by zjwang21** [thu-ml/ReMoE#1]
Have you made any comparisons when k>1? That is when k=2 or more for topk routers?

**Comment by thuwzt** [comment]
Yes, you can refer to the scaling experiments on granularity $G$. When $G=2$ TopK will select 2 experts out of the 16 available in our settings.
